build/zlib-1.2.8/contrib/vstudio/vc14/zlibstat.vcxproj

```diff
@@ -171,38 +171,11 @@
     <CodeAnalysisRules Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='ReleaseWithoutAsm|Win32'">
```

Comment:
Maybe I understood this part incorrectly, but i deleted it because it seemed that its only use was to define an alternate name for the target, which was not important.
The release configuration itself is kept.